src/components/v5/common/ActionSidebar/partials/Motions/steps/StakingStep/hooks.tsx

```diff
@@ -117,7 +115,7 @@ export const useStakingInformation = (
             },
           ),
           userProps: {
-            userName,
+            user,
```

Comment:
I'm unsure whether this was the right thing to do. Please let me know.

Reply:
I think it's fine